.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python:
+#  - 3.5
```

Comment:
Consider dropping 3.5 completely. I would remove the line instead of commenting out. You can always add it back in if needed.